cfgov/sheerlike/templates.py

```diff
@@ -8,8 +8,6 @@ def _convert_date(date, tz):
         if isinstance(date, basestring):
             date = parser.parse(date,
                                 default=datetime.datetime.today().replace(day=1))
-        # TODO: Event times are currently stored as UTC in the database, but they're actually ET times. 
-        # We will need to fix this once Wagtail fixes its handling of TIME_ZONE. 
         if type(date) in [datetime.datetime, datetime.date]:
             if isinstance(date, datetime.datetime):
                 pytzone = timezone(tz)
```

Comment:
At first I thought it was a good idea to keep it in because eventually we'll want to add the ability to localize times to a specific timezone. After thinking about it a little longer, this should probably go. Thanks for the heads up.